dev-util/sh/sh-3.5.1.ebuild

```diff
@@ -7,17 +7,40 @@ inherit go-module
 
 DESCRIPTION="A shell parser, formatter, and interpreter with bash support"
 HOMEPAGE="https://github.com/mvdan/sh"
-SRC_URI="https://github.com/mvdan/sh/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://dev.gentoo.org/~sam/distfiles/${CATEGORY}/${PN}/${P}-deps.tar.xz"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mvdan/sh.git"
+	RESTRICT="fetch mirror test"
```

Comment:
Please drop the RESTRICT. Tests work fine IIRC.

Also, please swap the order of EGIT_REPO_URI and inherit (sam style).